recipes/mintie/meta.yaml

```diff
@@ -9,12 +9,12 @@ source:
   sha256: 3de32e6b6a365f9b9e3469b3c8bfb5d3514c718386f0c32f49cc811547fef409
 
 build:
-  number: 0
+  number: 1
   noarch: generic
 
 requirements:
   run:
-    - bpipe
+    - bpipe =0.9.9.2
```

Comment:
Please add a comment that explains the issues or links to them.

Reply:
Thanks for reviewing. The original issue is described in https://github.com/Oshlack/MINTIE/issues/26. However, bpipe's author has mentioned incompatibility with JDK17, which is used for the bioconda bpipe builds from 0.9.9.7 onwards. 

I've submitted a pull request to fix this issue for bpipe: https://github.com/bioconda/bioconda-recipes/pull/38762. If this fixes the mintie issues, we can discard this PR.

Reply:
It looks like my PR for bpipe (https://github.com/bioconda/bioconda-recipes/pull/38762) is stuck. Would it be possible to approve this PR in the meantime to ensure MINTIE installs with a compatible version of bpipe?